package.json

```
@@ -12,7 +12,7 @@
   },
   "scripts": {
     "start": "node cli.js",
-    "test": "NODE_OPTIONS=\"--experimental-vm-modules --no-warnings\" jest --verbose --maxWorkers=1",
+    "test": "NODE_OPTIONS=\"--experimental-vm-modules --no-warnings\" jest --verbose",
```

```suggestion
    "test": "NODE_OPTIONS=\"--experimental-vm-modules --no-warnings\" jest --verbose --maxWorkers=1",
```

`--maxWorkers=1` is needed here because if multiple browsers run at the same time, their temp environments conflict with each other sometimes.